scripts/switch-submodule-branch.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
```

Comment:
Can you add a comment what does this script do?

Reply:
Done!